src/nativeMain/kotlin/com/github/h0tk3y/betterParse/lexer/com/github/h0tk3y/betterParse/lexer/RegexToken.kt

```diff
@@ -40,8 +37,7 @@ public actual class RegexToken : Token {
     }
 
     override fun match(input: CharSequence, fromIndex: Int): Int {
-        relativeInput.input = input
-        relativeInput.fromIndex = fromIndex
+        val relativeInput = RelativeInput(fromIndex, input)
```

Comment:
After this change, an instance of `RelativeInput` is created on each invocation of `match`, which, if not properly optimized, might cause a performance drop. I'd like to check how the benchmarks run on a native platform before and after the change.

Reply:
Hmm, `benchmarks` subproject does not seem to support native targets. Do you want me to add one?